public/data/shridharkapshikar.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Shridhar Kapshikar",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/test.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/shridharkapshikar",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/IamKShridhar",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "June 2022",
+      "icon": "dollar",
+      "color": "Orange",
+      "description": "Started freelancing",
```

Comment:
You now have a trailing comma on line 24 which is causing a couple of tests to fail. Deleting the comma should fix the issue.
```suggestion
      "description": "Started freelancing"
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Sorry for typo. 
I've the made the correction.